public/app/plugins/datasource/prometheus/components/PromQueryField.tsx

```diff
@@ -174,9 +174,6 @@ class PromQueryField extends React.PureComponent<PromQueryFieldProps, PromQueryF
   refreshMetrics = (cancelablePromise: CancelablePromise<any>) => {
     this.languageProviderInitializationPromise = cancelablePromise;
     this.languageProviderInitializationPromise.promise
-      .then(remaining => {
-        remaining.map((task: Promise<any>) => task.then(this.onUpdateLanguage).catch(() => {}));
-      })
```

Comment:
language provider's start function no longer returned any list of additional tasks. It's time to remove this task runner.

Reply:
I'd like to check if use this return argument anywhere and adapt the interface. I feel like language providers are doing too much work already.